dist/owncloud.js

```diff
@@ -1 +1 @@
"Basic "+Buffer.from("public:"+s).toString("base64")),i.Overwrite=n?"T":"F",i.Destination=a,this.davClient.request("MOVE",o,i,null,{version:"v2"}).then(e=>[200,201,204,207].indexOf(e.status)>-1?r.resolve(!0):r.reject(this.helpers.buildHttpErrorFromDavResponse(e.status,e.body)))}copy(e,t){let s=arguments.length>2&&void 0!==arguments[2]?arguments[2]:null,n=arguments.length>3&&void 0!==arguments[3]&&arguments[3];const i=this.helpers.buildHeaders(!1),o=this.getFileSharePath(e),a=this.getFileUrl(t);return s&&(i.Authorization="Basic "+Buffer.from("public:"+s).toString("base64")),i.Overwrite=n?"T":"F",i.Destination=a,this.davClient.request("COPY",o,i,null).then(e=>[200,201,204,207].indexOf(e.status)>-1?r.resolve(!0):r.reject(this.helpers.buildHttpErrorFromDavResponse(e.status,e.body)))}async getFileInfo(e){let t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:null,s=arguments.length>2&&void 0!==arguments[2]?arguments[2]:[];const r=await this.list(e,t,s,"0");return r[0]?r[0]:r}}},function(e,t,s){const r=s(36),n=s(0);e.exports=class{constructor(e){this.helpers=e}async getSettingsValues(){let e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:"me";try{const t=this.helpers.getInstance().replace(/\/$/,""),s=await r.ValueService_ListValues({client:this.helpers.axios,$domain:t,body:{account_uuid:e}});if(201===s.status)return n.resolve(s.data.values||[])}catch(e){if(e.response&&502!==e.response.status&&404!==e.response.status)return n.reject(e)}return n.resolve([])}}},function(e,t,s){"use strict";s.r(t),s.d(t,"getDomain",(function(){return h})),s.d(t,"setDomain",(function(){return l})),s.d(t,"request",(function(){return u})),s.d(t,"ValueService_ListValues",(function(){return c})),s.d(t,"ValueService_ListValues_RAW_URL",(function(){return p})),s.d(t,"ValueService_ListValues_TYPE",(function(){return d})),s.d(t,"ValueService_ListValuesURL",(function(){return f}));var r=s(3),n=s.n(r),i=s(4),o=s.n(i);let a="";const h=()=>a,l=e=>{a=e},u=(e,t,s,r,n,i,a)=>{t=t.toLowerCase();let h=s;return Object.keys(n).length>0&&(h=s+"?"+o.a.stringify(n)),r?e[t](h,r,a):"get"===t?e[t](h,a):e[t](h,o.a.stringify(i),a)},c=function(){let e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:{};const t=e.$domain?e.$domain:h(),s=e.$config,r=e.client||n.a;let i,o="/api/v0/settings/values-list",a={},l={};return void 0!==e.body&&(i=e.body),void 0===e.body?Promise.reject(new Error("Missing required  parameter: body")):(e.$queryParameters&&Object.keys(e.$queryParameters).forEach((function(t){a[t]=e.$queryParameters[t]})),u(r,"post",t+o,i,a,l,s))},p=function(){return"/api/v0/settings/values-list"},d=function(){return"post"},f=function(){let e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:{},t={};const s=e.$domain?e.$domain:h();let r="/api/v0/settings/values-list";e.$queryParameters&&Object.keys(e.$queryParameters).forEach((function(s){t[s]=e.$queryParameters[s]}));let n=Object.keys(t);return s+r+(n.length>0?"?"+n.map(e=>e+"="+encodeURIComponent(t[e])).join("&"):"")}}])}));
```

Comment:
this file should not be checked in

Reply:
It must to not have a release and use git url in package.json. Experimental specific only